.github/workflows/ci.yml

```diff
@@ -17,20 +17,30 @@ jobs:
       fail-fast: false
```

Comment:
Changes are probable best viewed with "Hide whitespace", since this introduces an `if` which changes existing script indentation.